<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QDialog" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit entrance in database</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/Images/computerIcon.png</normaloff>:/icons/Images/computerIcon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="8">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Edit Scientist or Computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QLineEdit" name="input_editPageSearch">
     <property name="placeholderText">
      <string>Search database</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="8">
    <widget class="QListWidget" name="list_editPageSearchResult"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_editScientist">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Scientist&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLabel" name="label_editScientistErrorMessage">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_editComputer">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5" colspan="3">
    <widget class="QLabel" name="label_editComputerErrorMessage">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLineEdit" name="input_editIdScientist">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>ID of Scientist you want to edit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="button_editScientist">
     <property name="text">
      <string>Edit Scientist</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QLineEdit" name="input_editIdComputer">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>ID of Computer you want to edit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6" colspan="2">
    <widget class="QPushButton" name="button_editComputer">
     <property name="text">
      <string>Edit Computer</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_editNameScientist">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="input_editNameScientist">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Edit name of Scientist</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="2">
    <widget class="QLabel" name="label_editNameComputer">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6" colspan="2">
    <widget class="QLineEdit" name="input_editNameComputer">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Edit name of Computer</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_editBirthYearScientist">
     <property name="text">
      <string>Birth year:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="input_editYearBorn">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Edit year born</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="2">
    <widget class="QLabel" name="label_editTypeComputer">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6" colspan="2">
    <widget class="QLineEdit" name="input_editType">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Edit type</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_gender">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Gender:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QComboBox" name="comboBox_editGender">
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="4" colspan="2">
    <widget class="QLabel" name="label_computerBuilt">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Was computer built?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="QComboBox" name="comboBox_editComputerBuilt">
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_ScientistAlive">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Is Scientist alive?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QComboBox" name="comboBox_editScientistAlive">
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="4" colspan="2">
    <widget class="QLabel" name="label_editYearOfCompletionComputer">
     <property name="text">
      <string>Year of completion:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6" colspan="2">
    <widget class="QLineEdit" name="input_editYearOfCompletion">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Edit year of completion</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_editDeathYearScientist">
     <property name="text">
      <string>Death year:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="input_editYearOfDeath">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Edit year of death</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QPushButton" name="button_saveEditScientist">
     <property name="text">
      <string>Save Scientist</string>
     </property>
    </widget>
   </item>
   <item row="10" column="4" colspan="4">
    <widget class="QPushButton" name="button_saveEditComputer">
     <property name="text">
      <string>Save Computer</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QPushButton" name="button_editBack">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>404</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="7">
    <widget class="QPushButton" name="button_editQuit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 0);
selection-color: rgb(170, 0, 0);
color: rgb(170, 0, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
